fn/func.go

```diff
@@ -130,6 +130,18 @@ func All[T any](xs []T, pred func(T) bool) bool {
 	return true
 }
 
+// AllMapItems returns true if the passed predicate returns true for all items
+// in the map.
+func AllMapItems[T any, K comparable](xs map[K]T, pred func(T) bool) bool {
```

Comment:
Cool! Looking forward to when/if they add more direct iterator stuff to the Go stdlib, then we'd only need one version of this.